### PR TITLE
MFW-3731: Backup Restore (add the logo image file in backup.tar)

### PR DESCRIPTION
**Pull request by sumedha-game**

### CODE REVIEW

**Review by manuwelakanade — APPROVED** [review]

File: backup-scripts/files/upload-backup.sh [hunk 1]
```diff
@@ -31,8 +31,14 @@ function createBackup() {
   debug "Backing up settings to gunzipped tar archive file"
   TEMP_DIR=`mktemp -d -t ut-backup.XXXXXXX`
   TEMP_DIR_NAME=$(basename $TEMP_DIR)
+  CP_DIR="/etc/config/captive_portal"
 
   cp /etc/config/settings.json $TEMP_DIR
+  if [ -d "$CP_DIR" ] ; then
+    cp $CP_DIR/* $TEMP_DIR
+    rm $TEMP_DIR/captive_portal_settings
```
Comment (manuwelakanade) [inline backup-scripts/files/upload-backup.sh:39]: Can we change this to rm -f $TEMP_DIR/captive_portal_settings

**Review by manuwelakanade — APPROVED** [review]

File: backup-scripts/files/upload-backup.sh (hunk 1 above)
Comment (sumedha-game) [reply backup-scripts/files/upload-backup.sh]: updated

File: backup-scripts/files/upload-backup.sh [hunk 2]
```diff
@@ -31,8 +31,14 @@ function createBackup() {
   debug "Backing up settings to gunzipped tar archive file"
   TEMP_DIR=`mktemp -d -t ut-backup.XXXXXXX`
   TEMP_DIR_NAME=$(basename $TEMP_DIR)
+  CP_DIR="/etc/config/captive_portal"
 
   cp /etc/config/settings.json $TEMP_DIR
+  if [ -d "$CP_DIR" ] ; then
```
Comment (manuwelakanade) [inline backup-scripts/files/upload-backup.sh:37]: Add a comment for why the directory check is here

Comment (sumedha-game) [reply backup-scripts/files/upload-backup.sh:37]: Directory check condition is for the older builds where we dont support captive portal, probably in version 6 of MFW we can remove this condition, will be in TODO list.

**Review by manuwelakanade — APPROVED** [review]

**Pull request merged**